listener.js

```diff
@@ -55,7 +59,10 @@ class Listener {
         this.stats         = {
             active_sessions:   0,
             accepted_sessions: 0,
-            counter: new StreamCounter()
+            counter: new StreamCounter(),
+            //Counter metric for total number of requests of the listeners
+            //can separate by dimensions(labels)
+            listener_requests_total: listener_requests_total
```

Comment:
Same stuff, can we collect all metrics relevant to listeners into a single class. We could add this to monitoring.js and then include it here, wdyt?

Reply:
Now instead of adding them one-by-one we just add a registry that stores all these metrics, you can get the reference of them out of this registry.